deploy/add_combine_field_values_function.psql

```diff
@@ -8,8 +8,23 @@ BEGIN;
 -- :"schema" variable - and default to public.
```

Comment:
Could we add some tests for the functions in this file using sample data from the live database?  Here's an example of similar tests we've done before: https://github.com/opensrp/opensrp-reveal-datawarehouse/blob/master/3-reveal/migrations/5-IRS/2-Zambia-2019/verify/process_structure_geo_hierarchy_queue.psql#L161

Reply:
Tests added [here](verify/store_all_for_multiple_choice_fields.psql).